src/base/bittorrent/session.cpp

```diff
@@ -346,6 +347,7 @@ Session::Session(QObject *parent)
     pack.set_int(libt::settings_pack::ssl_listen, 0);
     // To prevent ISPs from blocking seeding
     pack.set_bool(libt::settings_pack::lazy_bitfields, true);
+    pack.set_int(libt::settings_pack::alert_queue_size, 10000);
```

Comment:
> Did you have a better suggestion then torrent amount?

Why don't we reconfigure session (twice)? When starting up, set the queue size to accommodate alerts for all the torrents being loaded, and then configure it basing on the max connections number?